libsolidity/codegen/CompilerUtils.cpp

```diff
@@ -95,6 +95,26 @@ void CompilerUtils::revertWithStringData(Type const& _argumentType)
 	m_context << Instruction::REVERT;
 }
 
+void CompilerUtils::returnDataToArray()
+{
+	if (m_context.evmVersion().supportsReturndata())
+	{
+		m_context << Instruction::RETURNDATASIZE;
+		m_context.appendInlineAssembly(R"({
+			switch v case 0 {
```

Comment:
What's `v`?

Reply:
The topmost slot currently on the stack (see the last parameter of `appendInineAssembly`)

Reply:
Ah right